### DISCUSSION TITLE
Why excalidraw does not use standard, open-source file format/extension?

### DISCUSSION THREAD

**Discussion by rebelC0der** [excalidraw/excalidraw#6361] [General]
I was looking for an open-source, self-hosted MindMap tool, that uses some standard file formats.

I am wondering why most of the new and modern mindmap apps use custom file extensions like `.excalidraw`, and not `.mm` or even `.json` like some other mindmap apps?

Does anyone else have a concern, that they might build up a lot of work, using excalidraw, and one day, excalidraw becomes abandoned, and now you have no way of moving your work to another application? Why excalidraw has to use a custom file format?

Thanks.

**Comment by MintOcha** [comment]
open it in a text editor. it's Json with a different extension
